utilities/test_suite/README.md

```diff
@@ -0,0 +1,57 @@
+# AMD Radeon Performance Primitives (RPP) Test Suite
+The Radeon Performance Primitives library provides a test suite, that are used to validate the functionality and performance of the AMD Radeon Performance Primitives vision library. It can be executed under 2 backend scenarios - (HOST/HIP):
+- HOST backend - (On a CPU with HOST backend)
+- HIP backend - (On a GPU with HIP backend)
+
+## Features:
```

Comment:
Lets prioritize keeping "Modes of operation" above and move "Features" below it. In the "Features" section, lets edit the text now to just focus on - "What are the things you can do by running the test suite in any mode of operation?" For example - saying the test suite supports different layouts, bit depths, decoders, or the CSV dump or others.
As a last point you can add a NOTE saying QA mode currently supports only U8, or any other restrictions.

Reply:
Done